### DISCUSSION TITLE
app/How-to-use-WARP-on-Hiddify-App/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4587] [Content Help کمک به اصلاح محتوا]
# app/How-to-use-WARP-on-Hiddify-App/

The proxy manager platform.

https://hiddify.com/app/How-to-use-WARP-on-Hiddify-App/

**Comment by X-49** [comment]
What is the advantage of LQUIC over the familiar QUIC?
